GameDefinition/PLAYING.md

```diff
@@ -431,10 +431,22 @@ at least 5% of damage always gets through, even if the bonus is nominally
 below -95%, so excessively strong armor acts only as a buffer against
 high melee skill of opponents.
 
-The current pointman's melee bonus, armor modifier and other detailed
-stats can be viewed in the skill menu, accessible via the `#` command,
-which summarizes all the stats conferred by organs and conditions listed
-in the organ menu, invoked by `@`.
+The current pointsman's melee bonus (~d~ + % where '~' stands for a number),
```

Comment:
Later on we write "Weapon damage and other item properties are displayed using the dice notation `xdy`, which denotes `x` rolls of `y`-sided dice." Shall we standardize the notation? Either `~d~` in both places or `xyd` in both places? Or is this fine?

BTW, you can use "View file" from the right hand ... menu to see how the file is rendered in markdown. You'd notice that ~ without back quotes causes a text to be stroked through.

Reply:
I forgot the notation. I will change. BTW:
1. is it reasonable to write xdy + z%? Most of weapons is xd1. That means there are no dice.
2. z% - does it the bonus to this weapon? For example, ceramic splinter cells are 1d1 - 80%. Does it mean I need 5 splinters to inflict one point of damage? And does a spade 8d1 -- 10% inflict 7.2 points of damage?

Reply:
> is it reasonable to write xdy + z%?

Yes, I think so. Currently, `z` only appears in damage dice and then it's just percent of the rolled value added on top of it, as described in "The dice of the first recharged weapon, the one that is going to be used now, is adorned with percentage damage bonus collected from the whole equipment of the pointman". This is not well enough explained anywhere, so it makes sense to elaborate on that.

> Most of weapons is xd1. That means there are no dice.

Correct. Perhaps even all weapons. However, the engine supports weapons with non-trivial die, and also this resembles the common D&D and RPG notation, but underscores that there is no randomness, unlike in most other games.

> z% - does it mean the bonus to this weapon?

Yes, It affects the item it's written about. When displayed in item menu, it's only the percentage contributed by the weapon (or whatever the item is). But on HUD, it's the sum total that affects the weapon, summed from all equipment.

> For example, ceramic splinter cells are 1d1 - 80%. Does it mean I need to fling 5 cells to inflict one point of damage?

Yes. Thrown damage doesn't get bonuses from other equipment, so it will always be 80% for these splinters.

> And does a spade 8d1 -- 10% inflict 7.2 points of damage?

That's melee, but so if you wear nothing else, then 7.2, but if not, it can be different. The HUD displays that total. This is described somewhere, but without much detail, so worth discussion from a different angle and with different words.

Reply:
I mean, the sums of bonus are on HUD, but also in the # menu, as you correctly mention in the text.

Reply:
Another question:
A short club has 2d1 damage. But when the character equips it, it gives 2d1**+2** without percents. Does it mean that its damage was added to the character's natural one (I guess inflicted by the fists)?

Reply:
No, it's about different kinds of damage. That must be described in the manual. Could you have a look? If it's not easy to find, we have to make it so.

Reply:
BTW, that's also a key to an effective tank build, but the player is supposed to discover that by himself (as opposed to discovering UI meaning, which should be 100% clear and explicit from the docs).

Reply:
Hm. So far, I have not found anything similar in the "Battling monsters" section. I'll look at it again later.